includes/admin/plugins.php

```diff
@@ -398,6 +400,13 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			esc_html__( 'Visit plugin site', 'default' )
 		);
 	}
+
+	if ( $show_settings_link ) {
+		$plugin_link = perflab_get_plugin_settings_link( $plugin_data['slug'] );
+		if ( ! empty( $plugin_link ) ) {
```

Comment:
Let's avoid using `empty()`, for example:

* https://www.beberlei.de/post/when_to_use_empty_in_php_i_say_never
* https://dev.to/aleksikauppila/using-isset-and-empty-hurts-your-code-aaa

```suggestion
		if ( $plugin_link ) {
```